src/lib/Model/PublicModel.ts

```diff
@@ -531,6 +531,8 @@ export interface Cell {
     style?: CellStyle;
     /** Additional CSS classes */
     className?: string;
+    /** Checkbox cell specific properties */
+    checked?: boolean;
```

Comment:
We shouldn't add cell template specific properties to the base Cell interface...